protobuf-definitions/proto/mlagents_envs/communicator_objects/brain_parameters.proto

```diff
@@ -5,13 +5,21 @@ import "mlagents_envs/communicator_objects/space_type.proto";
 option csharp_namespace = "Unity.MLAgents.CommunicatorObjects";
 package communicator_objects;
 
+message ActionSpecProto {
+    int32 num_continuous_actions = 1;
+    int32 num_discrete_actions = 2;
+    repeated int32 discrete_branch_sizes = 3;
+    repeated string action_descriptions = 4;
+}
+
 message BrainParametersProto {
     reserved 1; // deprecated int32 vector_observation_size = 1;
     reserved 2; // deprecated int32 num_stacked_vector_observations = 2;
-    repeated int32 vector_action_size = 3;
+    repeated int32 vector_action_size_deprecated = 3; // mark as deprecated in communicator v0.22.0
```

Comment:
v1.3.0 (that's the one from https://github.com/Unity-Technologies/ml-agents/pull/4576/files)